frontend/src/component/project/Project/ProjectForm/ProjectForm.tsx

```diff
@@ -94,95 +81,28 @@ const StyledInputContainer = styled('div')(() => ({
     alignItems: 'center',
 }));
 
-const StyledFlagNamingContainer = styled('div')(({ theme }) => ({
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'flex-start',
-    gap: theme.spacing(1),
-    '& > *': { width: '100%' },
-}));
-
-const StyledPatternNamingExplanation = styled('div')(({ theme }) => ({
-    'p + p': { marginTop: theme.spacing(1) },
-}));
-
-export const validateFeatureNamingExample = ({
-    pattern,
-    example,
-    featureNamingPatternError,
-}: {
-    pattern: string;
-    example: string;
-    featureNamingPatternError: string | undefined;
-}): { state: 'valid' } | { state: 'invalid'; reason: string } => {
-    if (featureNamingPatternError || !example || !pattern) {
-        return { state: 'valid' };
-    } else if (example && pattern) {
-        const regex = new RegExp(`^${pattern}$`);
-        const matches = regex.test(example);
-        if (!matches) {
-            return { state: 'invalid', reason: 'Example does not match regex' };
-        } else {
-            return { state: 'valid' };
-        }
-    }
-    return { state: 'valid' };
-};
-
-const useFeatureNamePatternTracking = () => {
```

Comment:
This removes plausible tracking from project creation.

Reply:
FeatureNaming is not part of project creation

Reply:
Yes, you are correct.